src/framework/circuit.hpp

```diff
@@ -358,7 +365,9 @@ void Circuit::set_params(bool truncation) {
         case OpType::save_stabilizer:
         case OpType::save_unitary:
         case OpType::save_mps:
-        case OpType::save_superop: {
+        case OpType::save_superop:
+        case OpType::jump:
```

Comment:
I don't think jump and mark need to be here, since they dont produce output or change clbits themselves.